docs/snippets/ov_extensions.cpp

```diff
@@ -125,5 +152,12 @@ ov::Core core;
 core.add_extension("openvino_template_extension.so");
 //! [add_extension_lib]
 }
+
+{
+//! [frontend_extension_framework_map_macro_add_extension]
+ov::Core core;
+core.add_extension(ov::frontend::OpExtension<CustomOp>());
```

Comment:
It was designed differently. It should be enough to register it as `ov::OpExtension<CustomOp>` which should register it for both FW conversion and IR reading purposes. So the idea is to register regular OV op which has FW-extensions inside, and those extensions will be registered automatically while registering `CustomOp` as an OV op.

Reply:
@itikhono, is that correct?

Reply:
!!!

Reply:
This incorrect code still exists in documentation!